airbyte-ci/connectors/metadata_service/lib/tests/fixtures/metadata_upload/valid/metadata_base_image_exists.yaml

```diff
@@ -6,7 +6,7 @@ data:
     hosts:
       - zopim.com
   connectorBuildOptions:
-    baseImage: docker.io/airbyte/python-connector-base:1.1.0@sha256:bd98f6505c6764b1b5f99d3aedc23dfc9e9af631a62533f60eb32b1d3dbab20c
+    baseImage: docker.io/airbyte/base-repo-exists:1.1.0@sha256:shathatexists
```

Comment:
Changed all of these to the format we use to either indicate that the piece should exist or not. 

I think we should probably refactor the way we do this for both images and SHAs (and also probably docs) such that anything exists by default, unless `DOESNOTEXIST` or something similar is in it. That way, the files can look more like metadata and require less edits when we make new examples. But I'd do that later as it would touch a bunch of files.